### PR TITLE
Initial Implementation

### PR DESCRIPTION
**Pull request by scruplelesswizard**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM fedora:25
+
+# Install heketi-cli
+RUN ["/usr/bin/dnf","install","heketi-client","unzip","python","-y"]
+
+# Install kubectl
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/linux/amd64/kubectl /usr/local/bin/
+
+# Make kubectl executable
+RUN ["chmod","+x","/usr/local/bin/kubectl"]
```
Comment (scheeles) [inline Dockerfile:10]: Can you putt the `ADD` at the beginning of the Dockerfile and combine `RUN` e.g. https://github.com/docker-library/golang/blob/master/1.8/alpine/Dockerfile#L12
This reduce the size of the Dockerfile

**Review by scheeles — APPROVED** [review]

**Comment by scruplelesswizard** [comment]
Needs documentation added before merge

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # gk-installer
-Installer for gluster/gluster-kubernetes
+Installer for [gluster/gluster-kubernetes](https://github.com/gluster/gluster-kubernetes)
+
+## Usage
+The installer expects that you are running the container in the Kubernetes cluster you intend to deploy gluster-kubernetes to. It also expects that you will mount your topology file to /etc/gk-deploy/topology.json (i.e. via ConfigMap)
+
+The repo includes an example ConfigMap and Job for deployment
```
Comment (mrIncompetent) [inline README.md]: Please add a link to https://github.com/heketi/heketi/wiki/Setting-up-the-topology

File: gk-install-cm.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```
Comment (mrIncompetent) [inline gk-install-cm.yaml]: Should this file rather be moved into a "example" folder?

File: gk-install-job.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
```
Comment (mrIncompetent) [inline gk-install-job.yaml]: Should this file rather be moved into a "example" folder?

File: Dockerfile (hunk 1 above)
Comment (scruplelesswizard) [reply Dockerfile:10]: The container size difference is negligible, and in this case I would prefer the readability

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**